feature.env

```diff
@@ -1,6 +1,7 @@
 PORT=3007
 USE_OF_FORCE_URL=http://use-of-force
 PATHFINDER_URL=http://pathfinder
+API_PATHFINDER_URL=http://localhost:18080
```

Comment:
default is dev which we don't want for running feature tests locally.  There doesn't appear to be any stubbing of pathfinder apis at all in the integration tests so I've just hacked it to point at elite2 instead (and therefore return a 404)